deliverables/README.md

```diff
@@ -11,17 +15,39 @@ Sign Up for Cloud9 and create a basic Rails project.
 ###Intro to Ruby
 ####Required
 
-What is an "interpreted" language?<br>
-What is IRB?<br>
-What are Objects?<br>
-What are Methods?<br>
-What are Classes?<br>
-What are Blocks?<br>
-What is an Array?<br>
-What is an Iterator?<br>
-What are hashes?<br>
-What is a library?<br>
-What is a gem?
+**What is an "interpreted" language?**<br>
+It is a programming language that executes instructions directly, without first compiling a program into machine-language instructions.
+
+**What is IRB?**<br>
+Interactive Ruby Shell (IRB) is a REPL (read-eval-print loop) for programming in Ruby.
+
+**What are Objects?**<br>
+Objects, in general, are locations in memory that have a value and are referenced by an identifier. Objects can be variables, data structures, functions or even particular instances of a class where the object can be a combination of variables, functions and data structures.
+
+**What are Methods?**<br>
+Methods are procedures associated with object classes. They represent the behaviour of objects and are also the interface an object presents on the outside.
+
+**What are Classes?**<br>
+Classes are program code templates used to create objects, providing initial values for state and implementations of behaviour.
+
+**What are Blocks?**<br>
+Blocks are regions containing definitions of variables that delimit the regions where these definitions apply.
```

Comment:
And statements.  A block does create a scope for variables but can be characterized as a sequence of ruby commands/method calls.